web/src/cypress/integration/alerts.ts

```diff
@@ -378,7 +376,7 @@ function testAlerts(screen: ScreenFormat): void {
     beforeEach(() => {
       cy.createAlertLogs({ count: 200 }).then(_logs => {
         logs = _logs
-        return cy.visit(`/alerts/${logs.alert.id}`)
+        return cy.visit(`/alerts/${logs.alert.number}`)
```

Comment:
Was this causing a test failure due to the eslint changes?

Reply:
Currently looking into it, timepickers tests are also failing for some reason.

Reply:
The current schema doesn't include `alert.number`, probably related to the comment I left above:
```
type Alert {
  id: ID!
  alertID: Int!
  status: AlertStatus!
  summary: String!
  details: String!
  createdAt: ISOTimestamp!
  serviceID: ID!
  service: Service

  # Escalation Policy State for the alert.
  state: AlertState

  # Recent log entries for the alert.
  recentEvents(input: AlertRecentEventsOptions): AlertLogEntryConnection!
}
```